build/gulp-redcore/components/redcore.js

```diff
@@ -68,3 +82,14 @@ gulp.task('watch:' + baseTask + ':backend', function() {
 	{ interval: config.watchInterval },
 	['copy:' + baseTask + ':backend', browserSync.reload]);
 });
+
+// Watch frontend
+gulp.task('watch:' + baseTask + ':frontend', function() {
+	gulp.watch([
+		extPath + '/site/**/*',
+		extPath + '/../redcore.xml',
+		extPath + '/../install.php'
```

Comment:
you dont need to watch those two...
extPath + '/../redcore.xml',
extPath + '/../install.php'
it is watched by the backend....